src/main/webapp/i18n/de/result.json

```diff
@@ -55,9 +55,9 @@
             "successful": "Erfolgreich",
             "buildArtifact": "Build-Artefakt",
             "buildError": "Build Fehler:",
-            "score": "Bewertung {{score}}%",
+            "score": "Deine Bewertung ist {{score}}%",
```

Comment:
I think this is also used for the template and solution result, where "Deine" is not that fitting

Reply:
Did a `ctrl+f` search with `artemisApp.result.score` and only intended usages came up